cdk/tooling_environment/stacks/grafana/glue_crawlers_dashboard.template.json

```diff
@@ -112,7 +197,78 @@
   "schemaVersion": 38,
   "tags": [],
   "templating": {
-    "list": []
+    "list": [
+      {
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "hide": 0,
+        "includeAll": true,
+        "label": "Measure name",
+        "multi": true,
+        "name": "measure",
+        "query": "execution,failed,succeeded,duration_sec,dpu_seconds,tables_added,tables_updated,tables_deleted,partitions_added,partitions_updated,partitions_deleted",
```

Comment:
let's also have a quick demo during next sync-up to check how we show metrics like "tables_added" etc.